src/Stratis.Bitcoin.IntegrationTests/BlockStore/ReorgToLongestChainSteps.cs

```diff
@@ -83,7 +83,11 @@ private void bob_creates_a_transaction_and_broadcasts()
         {
             var nodeCReceivingAddress = this.GetSecondUnusedAddressToAvoidClashWithMiningAddress(this.nodes[Charlie]);
 
-            var transactionBuildContext = SharedSteps.CreateTransactionBuildContext(WalletZero, AccountZero, WalletPassword, nodeCReceivingAddress.ScriptPubKey, Money.COIN * 1, FeeType.Medium, minConfirmations: 1);
+            var transactionBuildContext = SharedSteps.CreateTransactionBuildContext(WalletZero
```

Comment:
I would also suggest that if it a new line per parameter, perhaps the first one should be too? Just an idea